editor/plugins/tiles/tile_map_editor.cpp

```diff
@@ -2214,12 +2214,14 @@ TileMapEditorTilesPlugin::TileMapEditorTilesPlugin() {
 	atlas_sources_split_container->add_child(tile_atlas_view);
 
 	tile_atlas_control = memnew(Control);
+	tile_atlas_control->set_anchors_and_offsets_preset(Control::PRESET_FULL_RECT);
```

Comment:
I believe this should instead be done inside the `add_control_over_alternative_tiles` and `add_control_over_alternative_tiles` functions. As it's kind of a requirement for any control to work there.